app/views/practices/_search.js.erb

```diff
@@ -1279,6 +1278,8 @@ function execSearchFunctions() {
     addAllCheckBoxListener('.all-clinical-checkbox', '.clinical-checkbox');
     addAllCheckBoxListener('.all-operational-checkbox', '.operational-checkbox');
     addAllCheckBoxListener('.all-strategic-checkbox', '.strategic-checkbox');
+    addAllCheckBoxListener('.all-communities-checkbox', '.communities-checkbox');
+    toggleMobilePracticePartnersAccordion();
```

Comment:
```suggestion
    toggleMobilePracticePartnersAccordion();
```
```suggestion
```
This was removed in #905, might need to rebase

Reply:
ah yea I did a rebase and had to go through and remediate the differences manually but I think I just missed that line, pushing up the change rn..